libs/styles/consonant-play-button.css

```
@@ -0,0 +1,113 @@
+
```

```suggestion

 :root {
  --play-icon-size-s: 44px;
  --play-icon-size-m: 77px;
  --play-icon-size-l: 102px;
}
```
What do you think if we set vars for these so they can change easily and are easier to read? I've got a hunch these values 'should' align w/ our icon-size definitions. This can be determined with a future consonant design discussion/task.

---

> What do you think if we set vars for these so they are more manageable? I've got a hunch these values 'should' align w/ our icon-size definitions. This can be determined with a future consonant design discussion/task.

@ryanmparrish I have made the requested changes and moved the sizes to variables. Since te overall size is computed using circle and border width, I have moved these in variable definitions as well.